src/vs/workbench/parts/debug/browser/debugEditorActions.ts

```diff
@@ -209,10 +211,97 @@ class ShowDebugHoverAction extends EditorAction {
 	}
 }
 
+class GoToBreakpointAction extends EditorAction {
+	constructor(private isNext, opts) {
+		super(opts);
+	}
+
+	public run(accessor: ServicesAccessor, editor: ICodeEditor, args: any): void | TPromise<void, any> {
+		const debugService = accessor.get(IDebugService);
+		const editorService = accessor.get(IEditorService);
+		const currentUri = editor.getModel().uri;
+		const currentLine = editor.getPosition().lineNumber;
+		const allEnabledBreakpoints =
+			debugService.getModel().getBreakpoints()
+				.filter(bp => bp.enabled)
```

Comment:
Instead of doing a .filter on top, please pass a "enabledOnly: true" to the `getBreakpoints`